ci/prow/make_config.go

```diff
@@ -916,6 +917,15 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 	executeJobTemplate("periodic", jobTemplate, title, repoName, data.PeriodicJobName, false, data)
 }
 
+// generateRemainingGoCoveragePeriodicJobs generate the go coverage periodic jobs for repositories that are not processed in the periodic job configuration
+func generateRemainingGoCoveragePeriodicJobs() {
```

Comment:
I think this approach is too specific. If we create a new kind of job that has implicit periodic jobs attached, this bug will manifest again. I would prefer a more generic approach that creates any required periodic jobs.

Reply:
BTW, it would be nice to use that approach also for postsubmit jobs. Now that we're confident about the generated config, it doesn't need to strictly follow the order of the original config file.